panedrlite/panedrlite/__init__.py

```diff
@@ -0,0 +1,7 @@
+# -*- coding: utf-8 -*-
+
+import pbr.version
+__version__ = pbr.version.VersionInfo('panedrlite').release_string()
```

Comment:
I see you updated this to fix the import issue that was happening. However the fact that CI wasn't picking up on the import error is pretty bad, we'll need to add some import tests (see the comment I made in the action yaml file).